interface/src/ui/Stats.h

```diff
@@ -248,9 +248,9 @@ private: \
  *     <em>Read-only.</em>
  * @property {string} lodStatus - Description of the current LOD.
  *     <em>Read-only.</em>
- * @property {string} numEntityUpdates - The number of entity updates that happened last frame.
+ * @property {number} numEntityUpdates - The number of entity updates that happened last frame.
```

Comment:
this (and some below) is unrelated cleanup to silence some warnings and fixup this JSDoc